cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -70,56 +155,17 @@
    Creates a mega menu primary navigation menu.
 
    ========================================================================== #}
-{% import '_vars-mega-menu.html' as vars with context %}
-<div class="o-mega-menu">
-    <button class="o-mega-menu_trigger">
-        <span class="o-mega-menu_trigger-label">
+<nav class="{{ base_class }}"
+     data-js-hook="flyout-menu"
+     aria-label="main navigation"
+     role="navigation">
+    <button class="{{ base_class ~ '_trigger' }}" data-js-hook="flyout-menu_trigger">
+        <span class="{{ base_class ~ '_trigger-label' }}">
```

Comment:
Ah, noop, this looks like it's an artifact of when the icon was attached to this span. Removed in https://github.com/cfpb/cfgov-refresh/commit/045d4d034e4a5b068113e835d4ea3828d2afa7f6